MobileBuy/sample/src/main/res/layout/customer_login_activity.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~   The MIT License (MIT)
```

Comment:
This comment is a good idea, we should probably add it to all our layouts in 2.0